test/lib.test.js

```diff
@@ -104,21 +104,6 @@ describe('uses the --prefix option to get the key', () => {
     const options = {
       imagemagick: useImageMagick,
       bucket: process.env.AWS_BUCKET,
-      profile: process.env.AWS_PROFILE
```

Comment:
There was a duplicate test here that had gotten committed previously and which I removed here.  We're not actually losing/changing any functionality with this change, Git's review is just making it look that way.